### ISSUE TITLE
[BUG] App search text issue

### ISSUE THREAD

**Issue by cfjedimaster** [PipedreamHQ/pipedream#661]
**Describe the bug**
I'm adding a step to a workflow and want to see the apps that exist under Google Calendar. I typed "google cal" in the filter and was surprised to not see it listed:

![image](https://user-images.githubusercontent.com/393660/101259629-903e0780-36ef-11eb-99a8-8a733dc9f8fa.png)

To be clear, if I kept scrolling down I could see it in the next section, oddly in *third* place after Google Sheets and Google Drive, but this was beneath the fold on my laptop. Maybe it does an "or" search on input? It seems like a phrase search would make more sense here.

**To Reproduce**
1. Go to a workflow
2. Click the plus to add an event
3. Type 'google cal' to filter your search

**Expected behavior**
I'd expect actions in the Google Calendar app to take priority.

**Desktop (please complete the following information):**
 - OS: Win 10
 - Browser: Edge

**Comment by cfjedimaster** [comment]
Actually, Google Calendar was *fourth* in the list of apps, after Google.

**Comment by dylburger** [comment]
@cfjedimaster yes, thank you for raising. Two word strings aren't currently handled well by our search algorithm. We'll look into this ASAP.

**Comment by dylburger** [comment]
Related to #1215
